### ISSUE TITLE
Not saving Banner record (hanging)

### ISSUE THREAD

**Issue by bojjaiah** [Magestore/Bannerslider-Magento2#27]
Hi,

I have installed in my local system for testing. It's hanging/looping while saving a record. See below Image half an hour, one hour still running.........

 
![banner slider](https://cloud.githubusercontent.com/assets/10813738/14134822/2f82d640-f677-11e5-846f-fc1b3ee8c134.png)

**Comment by alentsch** [comment]
Hi,

Same here! Installed the extension successfully and get the same "never ending" saving record. 

**system.log:**
`main.CRITICAL: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'magento.magestore_bannerslider_slider' doesn't exist, query was: SELECT COUNT(*) FROM `magestore_bannerslider_slider` AS `main_table` `

**Comment by jaatr** [comment]
run this command in cmd php bin/magento setup:static-content:deploy then refresh cache and index and then log out admin panel again login your problem will be solved

**Comment by jaatr** [comment]
run this command in cmd php bin/magento setup:static-content:deploy then refresh cache and index and then log out admin panel again login your problem will be solved

**Comment by bojjaiah** [comment]
@jaatr 

K. I will back after I tried.

**Comment by edenduong** [comment]
Hello,

The problem happened because the Banner Slider needs to deploy the JS/CSS to the pub/static folder. If your website is in the developer mode, the static content deploys automatically. If it is in the production mode, you should run the command:

php bin/magento setup:static-content:deploy

Regards,

**Comment by alentsch** [comment]
Hallo!

Thank you for your quick reply!
I tried the following commands:
```
$ magento setup:static-content:deploy
$ magento indexer:reindex
$ magento cache:flush
```
All commands are executed without error or warning. When I try to save a slider I do not get the endless rotating "saving record" symbol, but I get the following error message shown in the backend:
SQLSTATE[42S02]: Base table or view not found: 1146 Table 

> 'magento.magestore_bannerslider_slider' doesn't exist, query was: DESCRIBE `magestore_bannerslider_slider`
> Something went wrong while saving the slider.

I am attaching my var/log/system.log, which is showing some error trace.

Thank you so much for your help!!! 
Cheers!

**Comment by alentsch** [comment]
Update: I just upgraded to **Magento 2.0.4**, but unfortunately no change with the current issue. Still get the same error message :-(

**Comment by edenduong** [comment]
Hello,

I think the Banner Slider table in your database was not created. Could you please delete the Magestore_Bannerslider record in the setup_module table in your database and reinstall the module by the command line:

php bin/magento setup:upgrade

and try it again.

Regards,

**Comment by bojjaiah** [comment]
@edenduong 

Thank you for helping. I have fallowed the step by step and now working on newly released **Magento 2.0.4**.

**Comment by alentsch** [comment]
**Success!!** That worked for me, too! Thank you so much edenduong!!

PS: For those encountering this thread with a similar problem, and who are not so fit in mysql (like me), the commands I used are as follows:
```
(...) are optional commands

$ mysql -u root -p
> use magento;
( > show tables; )
( > show full columns from setup_module;  )
( > select * from setup_module;  ) 
> delete from setup_module where module = "Magestore_Bannerslider";
> exit;
```
